netpyne/sim/save.py

```diff
@@ -364,130 +364,59 @@ def intervalSave(simTime, gatherLFP=True):
     
     singleNodeVecs = ['t']
     simDataVecs = ['spkt','spkid','stims']+list(sim.cfg.recordTraces.keys())
-    if sim.nhosts > 1:
```

Comment:
The idea is to treat the case with no parallel context involved as if it is, but number of hosts is 1. This adds a tiny overhead of gathering data from nodes while there is effectively no nodes, but it seems to be neglectable as compared to code simplification and unification